libcontainer/standard_init_linux.go

```diff
@@ -86,15 +86,15 @@ func (l *linuxStandardInit) Init() error {
 	// initialises the labeling system
 	selinux.GetEnabled()
 
-	// We don't need the mountFds.SourceFds after prepareRootfs() nor if it fails.
+	// We don't need the mount nor idmap fds after prepareRootfs() nor if it fails.
 	err := prepareRootfs(l.pipe, l.config, l.mountFds)
-	for _, m := range l.mountFds.sourceFds {
+	for _, m := range append(l.mountFds.sourceFds, l.mountFds.idmapFds...) {
```

Comment:
This creates another slice. Not a big deal I guess.

Reply:
Closing this